custom-words.txt

```diff
@@ -1341,6 +1343,7 @@ mltable
 ML's
 mnist
 modelmanagement
+modsimworkbenches
```

Comment:
nit, is this needed? ever really used as all one word, or is it only used camelCase?

Reply:
validation failed on this and hence added here